src/app/component/ProductCard/ProductCard.component.js

```diff
@@ -74,6 +75,24 @@ export class ProductCard extends PureComponent {
 
     imageRef = createRef();
 
+    isConfigurableProductOutOfStock(productVariants) {
+        const variantsInStock = productVariants.filter((productVariant) => productVariant.stock_status === IN_STOCK);
+
+        return variantsInStock.length === 0;
+    }
+
+    isBundleProductOutOfStock(item) {
+        const { options } = item;
+
+        const optionsInStock = options.filter((option) => option.product.stock_status === 'IN_STOCK');
+
+        if (optionsInStock.length === 0) {
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
You have fixed the `isConfigurableProductOutOfStock` method, but forgot to fix the `isBundleProductOutOfStock` method.